### ISSUE TITLE
Website information section.

### ISSUE THREAD

**Issue by paaljoachim** [Really-Simple-Plugins/complianz-gdpr#330]
The Wizard - Website information.

![Screenshot 2021-12-31 at 11 08 24](https://user-images.githubusercontent.com/5323259/147817378-4e3918b3-6ea3-4c55-b3f0-cb41782401a7.jpg)

Owner of the website?
What is your address?
... and other fields.

The above fields will show up independent of selecting to have Complianz create a document or not.
Website information is really just about adding information to a possible created document which Complianz will create if the user added the option to do so. 

Bottom line is if user selects to have Complianz create a document then the Website information screen is needed. As have a GDPR page already created I would like to skip this page.

I do not see a way to skip page. I will instead need to click the next link in the left Wizard menu.

**Comment by Paapst** [comment]
This info is needed for creating a cookie policy (as it is a required part of the cookie banner).

**Comment by rlankhorst** [comment]
@paaljoachim @Paapst as we have the option to select your own cookie policy page, I agree with @paaljoachim that these questions should ideally not be visible if none of the documents where this is required is used. 

I'll see if this can be easily included in a generic way, as it is very basic info that is used in almost all documents we provide.

**Comment by rlankhorst** [comment]
https://github.com/Really-Simple-Plugins/complianz-gdpr/compare/conditions-if-no-documents-are-used?expand=1

**Comment by rlankhorst** [comment]
This will be released in 6.1, closing this for now.
